xcodeproj/internal/opts.bzl

```diff
@@ -134,12 +138,42 @@ def _get_unprocessed_compiler_opts(*, ctx, target):
     cpp = ctx.fragments.cpp
 
     return (
-        base_copts + cpp.copts + cpp.conlyopts + user_copts,
+        _flattened_unique_copts(base_copts, cpp.copts, cpp.conlyopts, objc_copts, user_copts),
```

Comment:
You can use the `uniq` function instead. It keeps the order and is more performant. M